src/infrastructure/rxUtils.ts

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright 2021 NEM
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { firstValueFrom } from 'rxjs';
+export const toPromise = firstValueFrom;
```

Comment:
why do we need this at all instead of just using the new name everywhere?

Reply:
the deprecated toPromise() method can be replaced toPromise with firstValueFrom or lastValueFrom.  Most of the time, like when calling and rest endpoint, the observable emits a single value. Either firstValueFrom or lastValueFrom should do the trick. Having said that, I didn't want to have one option placed everywhere so I used an alias. If we need to change it, we would just change the rxUtils.ts file.  Not 100% about this, maybe overthinking and less clear.

Reply:
@gimre-xymcity - what's your preference?

Reply:
> Most of the time, like when calling and rest endpoint, the observable emits a single value.

so I would just use firstValueFrom everywhere (the diff would also be slightly smaller and more readable)

on top of that, there's a chance that in future you'd want `firstValueFrom` in some places but `lastValueFrom` in others, so having this in a single place, doesn't really give any benefit.

Reply:
Thanks @gimre-xymcity , it makes perfect sense. I have removed the alias.